GoogleDataTransport/GDTCCTLibrary/GDTCCTUploadOperation.m

```diff
@@ -711,6 +702,39 @@ - (void)URLSession:(NSURLSession *)session
   }
 }
 
+#pragma mark - NSOperation methods
```

Comment:
Can we all these methods to a Category? GDTCCTUploader (NSOperation).

Mainly for separation of concerns.